Dockerfile

```
@@ -0,0 +1,36 @@
+ARG NODE_IMAGE_VERSION
+
+# Create base image
+FROM node:${NODE_IMAGE_VERSION} as base
+RUN apk add --no-cache tini python make g++
+
+# Install atlas engine
+FROM base as atlas_engine_fullstack_server
+
+ARG ATLAS_ENGINE_FULLSTACK_SERVER_VERSION
+
+COPY 'atlas_engine_fullstack_server_linux.tar.gz' ./
+RUN tar zxf atlas_engine_fullstack_server_linux.tar.gz
+RUN npm install sqlite3
+
+RUN npm link
+
+# Create release
+FROM atlas_engine_fullstack_server as release
+EXPOSE 8000
+ENTRYPOINT ["index.js"]
+
+VOLUME [ "/root/.config/atlas_engine_fullstack_server/" ]
+VOLUME [ "/usr/local/lib/node_modules/@atlas-engine/atlas_engine_fullstack_server/config/" ]
```

Mein Fehler, das npm paket heisst `@atlas-engine/fullstack_server`. Deshalb muss der Pfad hier lauten:

```suggestion
VOLUME [ "/usr/local/lib/node_modules/@atlas-engine/fullstack_server/config/" ]
```